### PR TITLE
Support both Ethereum and EVM family

### PR DESCRIPTION
**Pull request by JunichiSugiura**
fix https://github.com/LedgerHQ/ledger-live-issues/issues/108

### CODE REVIEW

File: packages/core/src/families/evm/serializer.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        maxPriorityFeePerGas: tx.maxPriorityFeePerGas.toString(),
+        maxFeePerGas: tx.maxFeePerGas.toString(),
+      }
+    : {
+        // ...baseTx,
+        gasLimit: tx.gasLimit.toString(),
+        data: tx.data?.toString("hex"),
+        additionalFee: tx.additionalFees?.toString(),
+        gasPrice: tx.gasPrice.toString(),
+      };
```
Comment (JunichiSugiura) [inline packages/core/src/families/evm/serializer.ts:40]: TODO: fix type error

File: packages/core/tests/serializers.spec.ts [hunk 2]
```diff
@@ -792,6 +809,22 @@ describe("serializers.ts", () => {
...
+      }
+
+      it("should succeed to deserialize an evm transaction without nullish/optional fields", () => {
+        const rawTx = createRawTx();
+        const tx = deserializeTransaction(rawTx);
+
+        expect(tx).toEqual({ ...rawTx });
+      });
+    });
+
```
Comment (JunichiSugiura) [inline packages/core/tests/serializers.spec.ts:827]: TODO: deserializer test

File: packages/core/src/families/evm/serializer.ts (hunk 1 above)
Comment (JunichiSugiura) [inline packages/core/src/families/evm/serializer.ts:40]: TODO: fix type error

File: packages/core/src/families/evm/serializer.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    : {
+        // ...baseTx,
+        gasLimit: rawTx.gasLimit ? BigNumber(rawTx.gasLimit) : undefined,
+        data: rawTx.data ? Buffer.from(rawTx.data) : undefined,
+        additionalFees: rawTx.additionalFees
+          ? BigNumber(rawTx.additionalFees)
+          : undefined,
+        gasPrice: BigNumber(rawTx.gasPrice),
+      };
+}
```
Comment (JunichiSugiura) [inline packages/core/src/families/evm/serializer.ts:73]: TODO: fix type error

File: packages/core/src/families/evm/types.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import type { z } from "zod";
+import type BigNumber from "bignumber.js";
+import type { TransactionCommon } from "..";
+import type {
+  schemaRawEvmTransaction,
+  schemaRawEvmTransactionEIP1559,
+} from "./validation";
+
+export interface EvmTransactionBase extends TransactionCommon {
```
Comment (JunichiSugiura) [inline packages/core/src/families/evm/types.ts:9]: Do we need to support `mode` field ? I don't see in our ethereum family. @IAmMorrow

**Review by JunichiSugiura — COMMENTED** [review]
Some note for next week.

**Comment by adrienlacombe-ledger** [comment]
@JunichiSugiura we close this one right?

**Pull request closed without merging**